chapter_appendix_math/statistics_vn.md

```diff
@@ -268,13 +268,13 @@ If a model suffers from high variance, we often say it is *overfitting* and lack
 ### Evaluating Estimators in Code
 -->
 
-### *dịch tiêu đề phía trên*
+### Đánh giá các Bộ ước lượng bằng Code
```

Comment:
```suggestion
### Đánh giá các Bộ ước lượng bằng mã
```

@tiepvupsu anh thấy "in code" mình dịch như thế này có ổn không? Nếu anh thấy ok thì để em đổi cái "Expression in code" lại luôn

Reply:
Cá nhân anh vẫn nghĩ nên dùng từ code vì ở VN hiếm nghe ai nói "mã" lập trình cả

Reply:
Anh đổi lại thành "qua lập trình" cho tự nhiên rồi nhé.